src/rng.js

```diff
@@ -1,7 +1,14 @@
 import crypto from 'crypto';
 
-const rnds8 = new Uint8Array(16);
+const poolSize = 4;
+const rnds8Pool = new Uint8Array(16 * poolSize);
+let poolPtr = -1;
 
 export default function rng() {
-  return crypto.randomFillSync(rnds8);
+  if (poolPtr === -1 || poolPtr === poolSize) {
+    crypto.randomFillSync(rnds8Pool);
+    poolPtr = 0;
+  }
+  const pos = 16 * poolPtr++;
+  return rnds8Pool.slice(pos, pos + 16);
```

Comment:
```suggestion
if (poolPtr > rnds8Pool.length - 16) {
  crypto.randomFillSync(rnds8Pool);
  poolPtr = 0;
}
return rnds8Pool.slice(poolPtr, poolPtr += 16);
```

Reply:
Applied the change in b73e4ab7da508afe5942101c601ffad61253a0ec